Misc/NEWS.d/next/Library/2021-12-14-13-15-41.bpo-45995.Am9pNL.rst

```diff
@@ -0,0 +1,3 @@
+Add "z" option to the string formatting specification, which coerces negative
+zero floating-point values to positive zero after rounding to the format precision.
+Covers ``float`` and ``Decimal`` formatting via :func:`str.format` and f-strings.
```

Comment:
Technically this covers anything that uses `float`, `decimal`, and `complex` `.__format__`, such as builtin `format()`. I'm not sure if it's best to just include `format()` here, or try to be more precise.

Reply:
Agreed (and we refined that description during the PEP).

I removed the offending text here, as it's probably too much detail for a news item anyway.